src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.IVsLanguageDebugInfo.cs

```diff
@@ -22,7 +22,7 @@ int IVsLanguageDebugInfo.GetLanguageID(IVsTextBuffer pBuffer, int iLine, int iCo
             {
                 return LanguageDebugInfo.GetLanguageID(pBuffer, iLine, iCol, out pguidLanguageID);
             }
-            catch (Exception e) when (FatalError.ReportAndCatch(e) && false)
```

Comment:
These were all doing ReportAndCatch because they wanted the "non fatal" behavior (which mind you was the case either way) but still wanted to propagate so that way the COM error handling still happens. We can call the regular method now.